src/V3Width.cpp

```diff
@@ -4846,6 +4890,33 @@ class WidthVisitor final : public VNVisitor {
         }
         userIterateChildren(nodep, nullptr);
     }
+    virtual void visit(AstWait* nodep) override {
+        if (v3Global.opt.timing() && nodep->fileline()->timingOn() && VN_IS(m_ftaskp, Func)) {
+            nodep->v3error("Wait statements are not legal in functions. Suggest use a task "
+                           "(IEEE 1800-2017 13.4.4)");
+            VL_DO_DANGLING(nodep->unlinkFrBack()->deleteTree(), nodep);
+            return;
+        }
+        if (nodep->fileline()->timingOn()) {
+            if (v3Global.opt.timing()) {
+                userIterateChildren(nodep, WidthVP(SELF, PRELIM).p());
+            } else {
+                nodep->v3warn(E_UNSUPPORTED, "Unsupported: wait statements\n"
+                                                 << nodep->warnMore()
+                                                 << "... Suggest use --timing");
```

Comment:
Again, it is supported, but only under '--timing'

Reply:
Fixed